lambda/link-scan/index.ts

```diff
@@ -0,0 +1,147 @@
+import { SendEmailCommand, SESv2Client } from '@aws-sdk/client-sesv2'
+import { Handler } from 'aws-cdk-lib/aws-lambda'
+import { LinkChecker } from 'linkinator'
+
+const SES_EMAIL_FROM = 'info@clearviction.org'
+const SES_EMAIL_TO = 'info@clearviction.org'
+
+export interface LinkData {
+    passed: boolean
+    linksScanned: number
+    brokenLinksCount: number
+    brokenLinks: {
+        url: string
+        parent?: string
+    }[]
+}
+
+const URL_TO_SCAN = 'http://clearviction.org'
+const REGION = 'us-west-2'
+
+export const handler: Handler = async () => {
+    try {
+        const returnData: LinkData = await checkLinks()
+
+        if (!returnData.passed) {
+            return await sendEmail(returnData)
+        }
+
+        return
+    } catch (error) {
+        return {
+            statusCode: 500,
+            headers: {
+                'Content-Type': 'application/json',
+                'Access-Control-Allow-Origin': '*',
+            },
+            body: JSON.stringify({
+                message: 'An error occurred while scanning the links',
+                error,
+            }),
+        }
+    }
+}
+
+export const checkLinks = async () => {
+    const checker = new LinkChecker()
+    const brokenLinks: LinkData['brokenLinks'] = []
+
+    checker.on('link', (result) => {
+        if (result.state === 'BROKEN') {
+            const newBrokenLink = {
+                url: result.url,
+                parent: result.parent,
+            }
+            brokenLinks.push(newBrokenLink)
+        }
+    })
+
+    const result = await checker.check({
+        path: URL_TO_SCAN,
+        recurse: true,
+    })
+
+    const brokenLinksCount = result.links.filter((x) => x.state === 'BROKEN')
```

Comment:
Just a little thing, what about change this `x` var name?
We could try make this more specific about on what we are talking about